theorems/T000274.md

```diff
@@ -5,12 +5,12 @@ if:
 then:
   P000133: true
 refs:
-  - doi: 10.1007/978-1-4612-6290-9_6
-    name: Counterexamples in Topology
+  - mathse: 3429751
+    name: Verification of basic properties of order topology
 ---
 
 Since discrete spaces are homeomorphic precisely when they have the same cardinality, it suffices to show that for every cardinality $k$ there is a set of cardinality $k$ with a total order that induces the discrete topology.
 
 Let $\kappa$ be a cardinal number. If $\kappa$ is finite, then any total order on $k$ induces the discrete topology. If $\kappa$ is infinite, let $X=\alpha \times \mathbb{Z}$ where $\alpha$ is an ordinal of cardinality $\kappa$, together with the lexicographical ordering. Then we see that $X$ with the order topology is discrete. 
 
-See example 39 of {{doi:10.1007/978-1-4612-6290-9_6}}.
+See example 39 of {{mathse:3429751}}.
```

Comment:
example 39 of that mathse reference doesn't make sense.

Reply:
Whoops, thanks. #143